frontend/components/TextArea/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./TextArea";
```

Comment:
Can you elaborate why you chose to add this file at the root of every components directory?